data/zone.schema.ts

```diff
@@ -0,0 +1,42 @@
+import mongoose from 'mongoose';
+import { Schema } from 'mongoose';
+
+enum Status {
+    todo = 'To Do',
+    done = 'Done',
+    inprogress = 'In Progress'
+}
+
+const zoneSchema = new Schema (
+    {
+        address: Object,
+        noUnsterilizedCats: {
```

Comment:
field might not be needed since a 'count' query on unsterilizedCats array will give us the information needed